pkg/actors/actors.go

```diff
@@ -1152,6 +1168,40 @@ func (a *actorsRuntime) doExecuteReminderOrTimer(ctx context.Context, reminder *
 }
 
 func (a *actorsRuntime) CreateReminder(ctx context.Context, req *CreateReminderRequest) error {
+	if a.scheduler != nil {
+		metadata := map[string]string{
+			"appId":        a.actorsConfig.AppID,
+			"actorType":    req.ActorType,
+			"actorId":      req.ActorID,
+			"reminder":     req.Name,
+			"content-type": "application/json",
+		}
+
+		jobName := constructCompositeKey(
```

Comment:
Do we want to standardize on the `"||"` or `_`. Personally, I think `_` is cleaner. I was adding job names as: `jobName := fmt.Sprintf("%s_%s", a.AppID(), inReq.GetJob().GetName())`

I was doing the `ListJobs()` based on the `appID` being the first string in the `jobName`, so these reminder jobs wont show up in the `ListJobs()` output as is.

Reply:
|| is the standard in Dapr.

Reply:
The reason is because "_" and many other commonly used separators are just too common and can conflict with ours. So, we picked one that is more rare and document it in our Dapr Docs, so applications know they need to avoid it.

If this is something anyone wants to change, it needs to be a proposal since it is a project-wide decision.